From d297e0dbb2ce8e58199f0db1c66267eab9568b8e Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Fri, 19 Jul 2019 00:08:55 +0200
Subject: [PATCH] strace: update to version 5.3

Makefile changes:
- moves PKG_MAINTAINER above PKG_LICENSE
- changes URL to a more appropriate one, which uses HTTPS
- adds 2 spaces as an indentation in description
---
 package/devel/strace/Makefile | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/package/devel/strace/Makefile b/package/devel/strace/Makefile
index 2d4332bf79..593b01feaf 100644
--- a/package/devel/strace/Makefile
+++ b/package/devel/strace/Makefile
@@ -9,19 +9,18 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk

 PKG_NAME:=strace
-PKG_VERSION:=5.0
+PKG_VERSION:=5.3
 PKG_RELEASE:=1

 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://strace.io/files/$(PKG_VERSION)
-PKG_HASH:=3b7ad77eb2b81dc6078046a9cc56eed5242b67b63748e7fc28f7c2daf4e647da
+PKG_HASH:=6c131198749656401fe3efd6b4b16a07ea867e8f530867ceae8930bbc937a047

+PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 PKG_LICENSE:=BSD-3c
 PKG_LICENSE_FILES:=COPYRIGHT
 PKG_CPE_ID:=cpe:/a:paul_kranenburg:strace

-PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
-
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1

@@ -47,13 +46,13 @@ define Package/strace
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=System call tracer
+  URL:=https://strace.io/
   DEPENDS:=+STRACE_LIBDW:libdw +STRACE_LIBUNWIND:libunwind
-  URL:=http://strace.sourceforge.net/
 endef

 define Package/strace/description
-A useful diagnostic, instructional, and debugging tool. Allows you to track what
-system calls a program makes while it is running.
+  A useful diagnostic, instructional, and debugging tool. Allows you to track what
+  system calls a program makes while it is running.
 endef

 define Package/strace/config
--
2.20.1
